### PR TITLE
Added git-lfs to the build script to better support no-arg builds of gatk

### PR DESCRIPTION
**Pull request by jamesemery**
Modifies #6764

### CODE REVIEW

**Review by KevinCLydon — DISMISSED** [review]
Looks good to me 👍

File: Dockerfile [hunk 1]
```diff
@@ -6,6 +6,12 @@ RUN ls .
 ADD . /gatk
 WORKDIR /gatk
 
+RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
```
Comment (droazen) [inline Dockerfile:9]: `git-lfs` is in the official package repositories for both ubuntu 18.04 and ubuntu 20.04 -- are you sure that you need this step?

**Comment by KevinCLydon** [comment]
Without the curl command, gcloud build failed with this error message: The command '/bin/sh -c apt-get --assume-yes install git-lfs' returned a non-zero code: 100

**Comment by droazen** [comment]
@KevinCLydon @jamesemery That's probably because you need to activate the ubuntu "universe" repository in order to install the package. This can be done via the command `add-apt-repository universe && apt update`, I believe.

**Comment by KevinCLydon** [comment]
Building works for the latest commit

**Comment by jamesemery** [comment]
@KevinCLydon if it works can I get the 👍 from you so we can merge this?

**Pull request merged**